Add option to use system projectm-eval (used in vcpkg) (#803)
projectM-visualizer/projectm: CMakeLists.txt

option(ENABLE_BOOST_FILESYSTEM "Force the use of boost::filesystem, even if the compiler supports C++17." OFF)
cmake_dependent_option(ENABLE_INSTALL "Enable installing projectM libraries and headers." OFF "NOT PROJECT_IS_TOP_LEVEL" ON)
option(ENABLE_SYSTEM_GLM "Enable use of system-install GLM library" OFF)

> option(ENABLE_SYSTEM_PROJECTM_EVAL "Enable use of system-install projectM-eval library" ON)

Reviewer: No one will read this text anyway, but I'd suggest a slight change:
```suggestion
option(ENABLE_SYSTEM_PROJECTM_EVAL "Enable use of a system-installed/external projectM-eval library" ON)
```

> option(ENABLE_SYSTEM_PROJECTM_EVAL "Enable use of system-install projectM-eval library" OFF)

Reviewer: I'd actually prefer this if it's available. Linux distro s like Gentoo also ideally want this as separate package. So the build script should just look for it by default, giving the dev the opportunity to disable it and use the built-in version.

```suggestion
option(ENABLE_SYSTEM_PROJECTM_EVAL "Enable use of system-install projectM-eval library" ON)
```

    find_package(GLM REQUIRED)
endif()

if(ENABLE_SYSTEM_PROJECTM_EVAL)

>     find_package(projectM-eval REQUIRED)

Reviewer: Remove the REQUIRED keyword and use correct casing for the config package dir:

```suggestion
    find_package(projectM-Eval)
```
